Models/Project/DonutShop.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace rexfinder_api.Models;
+
+public class DonutShop
+{
+    [Required]
+    public int DonutShopId { get; set; }
+    public string DonutShopName { get; set; }
+    public string DonutShopAddress { get; set; }
+    public string DonutShopWebsite { get; set; }
+}
```

Comment:
Do we want to add a photoURL to be able to show a photo?

I'm wondering if we may also want to add a UserId / Users to this model but make it optional? That way, maybe there's another part of the easter egg where users can add their favorite donut shop to the list? That would definitely be stepping into V2 territory, but I think it could be cool to add, especially since I don't think we'll be able to get a lot of V2 features done.